src/stdune/path.ml

```diff
@@ -431,6 +431,9 @@ let readdir t = Sys.readdir (to_string t) |> Array.to_list
 let is_directory t =
   try Sys.is_directory (to_string t)
   with Sys_error _ -> false
+let is_file t =
```

Comment:
`is_file fn` makes me think `not (is_directory fn)` while  this is not the case. Why not simply call the function `file_exists`?

Reply:
Sure, though shouldn't `Path.exists` (which is the same as Sys.file_exists) catch Sys_error?

Reply:
Indeed, catching Sys_error in Path.exists seems fine